Modules/Salary/Resources/assets/js/components/SalaryBreakdown.vue

```diff
@@ -46,6 +48,122 @@
 				</div>
 			</div>
 		</div>
+		<h2 class="text-underline">Salary Deduction</h2>
+		<br>
+		<div class="row mb-5">
+			<div class="col-md-4">
+				<div class="text-secondary mb-1">Employee ESI</div>
+				<div class="fz-30">
+					<i class="fa fa-rupee"></i>
+					<span>NA</span>
```

Comment:
please fix at all places
```suggestion
					<span>N/A</span>
```